aggregate_root/.mutant.yml

```diff
@@ -0,0 +1,11 @@
+---
+coverage_criteria:
+  # Control the timeout criteria, defaults to `false`:
+   # * `true` - mutant will consider timeouts as covering mutations.
+   # * `false` mutant will ignore timeouts when determining mutation coverage.
+   timeout: true
```

Comment:
@pawelpacana Make sure to also configure a timeout, else this setting has no effect.

Reply:
https://github.com/RailsEventStore/rails_event_store/pull/840/commits/23ac8ea6bc95ee7460c1af1b8263bf0cab1e3f76

Reply:
thanks. BTW I see you run a full pass on CI, this is not recommended, normally you should only run incremental passes.

Also 3/s is a very low speed. I'll have a look at your setup.

Reply:
How does incremental work when mutant is getting new or improved transformations?

Reply:
> How does incremental work when mutant is getting new or improved transformations?

it would only select changed code, even if mutant gained new operators. Incremental does not replace running full runs once in a while.

Still for bigger projects its recommended. But: As I meanwhile discussed this project has very fast tests, hence you should get away with full runs.

I'll do some optimiaztions in PRs, thanks for getting me access so I can try with the CI integration.